hyperjumptech/monika
# added: insomnia support

# Monika Pull Request (PR)
This PR resolves #556

## What feature/issue does this PR add
1. Added: Insomnia collection exported file support in JSON & YAML format
2. Limit export format in form of v4

## How did you implement / how did you fix it
1. Added new flag `--insomnia/-I`
2. Added new file `src/components/config/parse-insomnia.ts`
3. Added new test to `src/components/config/__tests__/create-config.test.ts`
4. Added sample export file `src/components/config/__tests__/petstore.insomnia.yaml`
5. Added expected generated file `src/components/config/__tests__/expected.insomnia.yml`

## How to test
1. Run from sample file: `npm run start -- --verbose --insomnia src/components/config/__tests__/petstore.insomnia.yaml`
2. Verify probe config
3. Run the test: `npm run test`
